testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/toolbarIntegration.tests.js

```diff
@@ -613,6 +613,67 @@ export default function() {
             $okDialogButton.trigger('dxclick');
         });
 
+        test('href in markup should be empty when empty href is passed in value (T1134100)', function(assert) {
+            const $container = $('#htmlEditor').dxHtmlEditor({
+                value: '<a href="">test</a>',
+            });
+            const linkHref = $container.find('a').attr('href');
+
+            assert.strictEqual(linkHref, '')
+        });
+
+        test('href should be empty on empty URL input submit (T1134100)', function(assert) {
+            const done = assert.async();
+            const $container = $('#htmlEditor');
```

Comment:
I suggest u to name it more strict everywhere :)
```suggestion
            const $htmlEditor = $('#htmlEditor');
```